autogluon/utils/tabular/ml/learner/abstract_learner.py

```diff
@@ -275,7 +275,7 @@ def score_debug(self, X: DataFrame, y=None, compute_oracle=False, silent=False):
 
         for model_name, pred_proba in model_pred_proba_dict.items():
             if (trainer.problem_type == BINARY) and (self.problem_type == MULTICLASS):
-                pred_proba = self.label_cleaner.inverse_transform_proba(pred_proba)
+                pred_proba = self.label_cleaner.inverse_transform_proba(pred_proba)  # FIXME: I think this doesn't work correctly, must use original y as well!
```

Comment:
No because it only occurs in the extreme edge case of multiclass with only 2 frequent classes, which we have yet to have a real dataset that does this.